lib/AfricasTalking/Token.rb

```diff
@@ -1,66 +1,66 @@
+# frozen_string_literal: true
+
 class Token
```

Comment:
to me all these classes should rather be in the AfricasTalking module, i.e.
```rb
module AfricasTalking
  class Token
    # ...
  end
end
```

So we would have a proper encapsulation of the library, with classes as `AfricasTalking::Token`, instead of "polluting" the Ruby global namespace with things that are AfricasTalking-specific but using generic names that a user application could be using already - `Token`, `Sms`, `StatusCode` are all quite generic concepts, and a gem should probably not use them without embedding them in a proper dedicated module? :slightly_smiling_face: